src/orange/OrangeTrackView.hh

```diff
@@ -234,15 +249,18 @@ CELER_FUNCTION
 OrangeTrackView& OrangeTrackView::operator=(DetailedInitializer const& init)
 {
     CELER_EXPECT(is_soft_unit_vector(init.dir));
-    CELER_EXPECT(states_.vol[init.other.thread_]);
+
+    for (auto i : range(states_.level[init.other.thread_] + 1))
+    {
+        // Copy all data accessed via LSA, except dir, which is set via init
+        auto lsa = this->make_lsa(LevelId{i});
+        lsa = init.other.make_lsa(LevelId{i});
+        lsa.dir() = init.dir;
```

Comment:
This will only be true if don't support rotations. It might be safer to copy all the levels and then call `this->set_dir(init.dir)`?

Reply:
Do you mean rotations of universes? I don't understand why `set_dir` is relevant, as it looks like its primary purpose is boundary checking. It looks like right now calling `set_dir` breaks the GEANT4 tests, so this is something else I vote we punt on.

Reply:
Yeah, rotations of universes. Once we have rotations in, then the direction will need to be updated at the lower levels. We can punt.

Reply:
(Unresolving this conversation since it's something for future work: unlike code-int we don't have to resolve everything before merging.)